src/index.js

```diff
@@ -1638,6 +1639,9 @@ export default function ({types: t, template}): Object {
         else if (isPolymorphicType(annotation.id, scope)) {
           return;
         }
+        else if (/^HTML.*Element$/.test(annotation.id.name)) {
```

Comment:
This is a bit too broad, it will match the DOM's builtin elements which have nothing to do with React. In fact, react elements are `React.Element`s so this will not interop with flow.